### PR TITLE
add region to Serbia

### PR DESCRIPTION
**Pull request by Nauna**
fixes # 1
https://nanopay.atlassian.net/browse/NP-5377

### CODE REVIEW

**Comment by jlhughes** [comment]
LGTM

**Review by jlhughes — APPROVED** [review]

**Review by lchanmann — APPROVED** [review]

File: src/regions [hunk 1]
```diff
@@ -564,6 +564,37 @@ p({"class":"foam.nanos.auth.Region", "code":"PT-16", "isoCode":"16", "name":"Via
 p({"class":"foam.nanos.auth.Region", "code":"PT-20", "isoCode":"20", "name":"Azores", "countryId":"PT"})
 p({"class":"foam.nanos.auth.Region", "code":"PT-30", "isoCode":"30", "name":"Madeira", "countryId":"PT"})
 
+// Serbia
+p({"class":"foam.nanos.auth.Region", "code":"RS-01", "isoCode":"01", "name": "Severnobački okrug", "countryId":"RS"})
+p({"class":"foam.nanos.auth.Region", "code":"RS-02", "isoCode":"02", "name": "Srednjebanatski okrug", "countryId":"RS"})
```
Comment (lchanmann) [inline src/regions:569]: @Nauna, I noticed "okrug" is repeating for all regions, might be something we don't need to store?

Comment (jlhughes) [reply src/regions:569]: The logic for that would be more than than the few lines of translations we have.   If this occurs again for other countries, let's consider this again.

Comment (lchanmann) [reply src/regions:569]: "okrug" in Serbian is "district".

Comment (Nauna) [reply src/regions:569]: Yes district.
So copied the names from https://www.iso.org/obp/ui/#iso:code:3166:RS and assumed the language of the country made the most sense until we have translations for regions

The names in the AFEX response wasn't very reliable:
example:
Id": "14", "Name": "Bor",
"Id": "26", "Name": "Pec´",

**Pull request merged**